modules/olm-operatorhub-architecture.adoc

```diff
@@ -8,23 +8,16 @@
 The OperatorHub UI component is driven by the Marketplace Operator by default on
 {product-title} in the `openshift-marketplace` namespace.
 
-The Marketplace Operator manages OperatorHub and OperatorSource Custom Resource
-Definitions (CRDs).
-
-[NOTE]
-====
-Although some OperatorSource information is exposed through OperatorHub's UI
-interface, it is only used directly by those who are creating their own
-Operators.
-====
+The Marketplace Operator manages the OperatorHub custom resource
+definition (CRD).
 
```

Comment:
```suggestion
The Marketplace operator manages the OperatorHub CustomResource named `cluster` 
```
There can be other OperatorHub CRs too, but marketplace won't respond to changes in them. The OperatorHub CRD is not managed by marketplace